Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesEF.swift

```diff
@@ -3806,23 +3798,17 @@ public struct FunctionEffectSpecifiersSyntax: SyntaxProtocol, SyntaxHashable, _L
     }
   }
   
-  /// The `throws` or `rethrows` keyword.
-  ///
-  /// ### Tokens
-  /// 
-  /// For syntax trees generated by the parser, this is guaranteed to be one of the following kinds:
-  ///  - `throws`
-  ///  - `rethrows`
-  public var throwsSpecifier: TokenSyntax? {
+  /// The clause specifying thrown errors
+  public var throwsClause: ThrowsClauseSyntax? {
     get {
-      return Syntax(self).child(at: 3)?.cast(TokenSyntax.self)
+      return Syntax(self).child(at: 3)?.cast(ThrowsClauseSyntax.self)
     }
     set(value) {
       self = Syntax(self).replacingChild(at: 3, with: Syntax(value), arena: SyntaxArena()).cast(FunctionEffectSpecifiersSyntax.self)
     }
   }
   
-  public var unexpectedBetweenThrowsSpecifierAndThrownError: UnexpectedNodesSyntax? {
+  public var unexpectedAfterThrowsClause: UnexpectedNodesSyntax? {
```

Comment:
This needs a compatibility layer.

Reply:
The rule that we are currently following is that two swift-syntax versions should be API-compatible except for the incompatibilities noted in the release notes. And as the unexpected fields are part of the public API, I think they should follow the same rules – even if we don’t expect them to be used much

Reply:
I opened https://github.com/apple/swift-syntax/pull/2393 to add the compatibility  layers.